proto/build.gradle.kts

```diff
@@ -0,0 +1,70 @@
+import de.undercouch.gradle.tasks.download.Download
+import de.undercouch.gradle.tasks.download.Verify
+
+plugins {
+    id("java-library")
+    id("maven-publish")
+
+    id("com.google.protobuf")
+    id("de.undercouch.download")
+    id("ru.vyarus.animalsniffer")
+}
+
+description = "OpenTelemetry Proto"
+extra["moduleName"] = "io.opentelemetry.proto"
+
+dependencies {
+    api("com.google.protobuf:protobuf-java")
+    api("io.grpc:grpc-api")
+    api("io.grpc:grpc-protobuf")
+    api("io.grpc:grpc-stub")
+}
+
+val protoVersion = "0.7.0"
+// To generate checksum, download the file and run "shasum -a 256 ~/path/to/vfoo.zip"
+val protoChecksum = "0b581c654b2360485b99c2de3731dd59275b0fe7b91d78e7f6c5efd5997f4c82"
+val protoArchivePath = "$buildDir/archives/opentelemetry-proto-${protoVersion}.zip"
```

Comment:
Could we make it a `file` here already? Does `zipTree(file)` work?

Reply:
Thanks, it does